packages/loader/container-loader/src/containerContext.ts

```diff
@@ -85,7 +85,12 @@ export class ContainerContext implements IContainerContext {
 			batch: IBatchMessage[],
 			referenceSequenceNumber?: number,
 		) => number,
-		public readonly submitSignalFn: (content: ISignalEnvelope, targetClientId?: string) => void,
+		// `unknown` should be removed once @alpha tag is removed from IContainerContext
```

Comment:
Work item should be filed and there should be a comment placed with IContainerContext's `@alpha` tag so that when touched people will see the requested follow-up.